Minesweeper.py

```diff
@@ -178,91 +170,82 @@ def get_remaining_flags(self):
         return count
 
     def loop(self):
+        """ handles the main game loop
+        """
         run = True
         while run:
 
-            # DHB: use a helper function for keyboard events, rather than mixing it into the main loop
```

Comment:
still haven't addressed this comment. Please do not remove my comments unless you address them.